packages/generator/CHANGELOG.md

```diff
@@ -1,5 +1,88 @@
 ## CHANGELOG
 
+### v1.3.0-alpha.8
+
+### **Core Functionality**
+
+#### Helpers
+
+* **BREAKING**: helpers.getAuthToken method is replaced by helpers.getShopperToken
+
+#### Client Configuration
+
+* helpers.getShopperToken supports Client Configuration. Refer to [Sample Code](./README.md#Sample Code)
```

Comment:
This and the above can both be in the same subheading. Instead of "Helpers" I would call that heading "Authentication helper changes"